src/api/controllers/password.reset.js

```diff
@@ -33,7 +32,7 @@ export default class PasswordReset {
       userName: username,
       buttonText: 'RESET',
       message:
-        "You are receiving this email beacause you've requested the recovery "
+        "You are receiving this email because you've requested the recovery "
```

Comment:
Why are you not using the error message sender here?  consider sending the message using it to get the right format

Reply:
already solved